src/azure/Templates/AzureServiceClientOperationsTemplateAsync.cshtml

```diff
@@ -13,6 +13,11 @@
 @Header("# ").TrimMultilineHeader()
 # --------------------------------------------------------------------------
 @EmptyLine
+@if (Model.AddTracingDecorators)
+{
+@:from azure.core.tracing.decorator import distributed_trace
```

Comment:
This is the async file, do you need this?

Reply:
for keyvault there are both sync and async methods to trace in the operations_async folder. The `import_key` is async but the `list_keys` is sync.